components/sync/service/brave_sync_service_impl.h

```diff
@@ -61,6 +61,7 @@ class BraveSyncServiceImpl : public SyncServiceImpl {
   void Initialize() override;
 
   const brave_sync::Prefs& prefs() { return brave_sync_prefs_; }
+  brave_sync::Prefs& modifying_prefs() { return brave_sync_prefs_; }
```

Comment:
This can just be `prefs()`, compiler will figure out the right version for you.
Can we also update the above function to be `const brave_sync::Prefs& prefs() const`;

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/21767/commits/d5b623d8846cb44a00af13cabdf07ba7b8c5fcd8